src/components/meeting/meetingTime/Meeting.js

```diff
@@ -26,16 +26,16 @@ export const Meeting = () => {
         <div>
             {
                 isFetching ? <CircularProgress /> :
-                <Fetched name={name} interviewId={interviewId} />
+                <Fetched name={name} interviewId={interviewId} companyId={companyId} />
             }
         </div>
     )
 }
 
-const Fetched = ({ name, interviewId }) => {
+const Fetched = ({ name, interviewId, companyId }) => {
     return (
         name === null ?
```

Comment:
Tutaj chyba `==` byłoby lepsze

Reply:
Nie wiem czemu - domyślnie ustawiam na null, więc chcę wiedzieć, kiedy jest dokładnie nullem

Reply:
Prawdą to jest, ale nigdy nie wiesz, kiedy nagle `name` stanie się `undefined`, a to jak wiadomo może się w świecie JSa zdarzyć, no i wtedy nie zadziała